src/bin/rav1e.rs

```diff
@@ -196,11 +192,11 @@ fn main() {
 
   if video_info.bit_depth == 8 {
     do_encode::<u8, y4m::Decoder<'_, Box<dyn Read>>>(
-      cfg, cli.verbose, progress, err, &mut cli.io.output, &mut source, y4m_enc
+      cfg, cli.verbose, progress, err, &mut *cli.io.output, &mut source, y4m_enc
```

Comment:
This probably will need a manual rebase.

Reply:
Let see if I managed to do it directly from the github UI

Reply:
No, it failed, I manually redid properly

Reply:
Thanks for help.
Updated.